scripts/linters/python_linter_test.py

```diff
@@ -150,7 +150,7 @@ def test_invalid_file_with_pylint_error(self):
             python_linter.ThirdPartyPythonLintChecksManager(
                 [INVALID_DOCSTRING_FILEPATH], True).perform_all_lint_checks()
         self.assert_same_list_elements(
-            ['C: 28, 0: Period is not used at the end of the docstring.'],
+            ['W: 27, 0: Period is not used at the end of the docstring.'],
```

Comment:
Could you explain why this changed?

Reply:
I changed this because we are using `node` instead of the `line number` in the messages.
Before:
`self.add_message('newline-below-class-docstring', line_num=line_number)`

Now:
`self.add_message('newline-below-class-docstring', node=node)`

This will give the line number from where the node starts.

Reply:
I see, thanks for the explanation.